### PR TITLE
Support icon variant on commands, make start and stop icons filled

### PR DESCRIPTION
**Pull request by JamesNK**
## Description

Consistent feedback on the start/stop/restart icons was the stop icon looked like an empty checkbox. The standard for this kind of UI is to have filled icons. For example, VS debugging start/stop buttons are filled.

This PR:

* Adds icon variant option to commands. Defaults to regular.
* Changes start and stop icons to be filled icons.
* Changes the icon size from 20 to 16. Size 20 filled icons felt too big. Size 16 looks good.
* Fixes error when command doesn't have an icon.

Demo:
image

## Checklist

- Is this feature complete?
  - [x] Yes. Ready to ship.
  - [ ] No. Follow-up changes expected.
- Are you including unit tests for the changes and scenario tests if relevant?
  - [ ] Yes
  - [x] No
- Did you add public API?
  - [x] Yes
    - If yes, did you have an API Review for it?
      - [ ] Yes
      - [ ] No
    - Did you add `<remarks />` and `<code />` elements on your triple slash comments?
      - [ ] Yes
      - [ ] No
  - [ ] No
- Does the change make any security assumptions or guarantees?
  - [ ] Yes
    - If yes, have you done a threat model and had a security review?
      - [ ] Yes
      - [ ] No
  - [x] No
- Does the change require an update in our Aspire docs?
  - [ ] Yes
    - Link to aspire-docs issue: 
  - [x] No

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/dotnet/aspire/pull/5839)

### CODE REVIEW

File: src/Aspire.Hosting/Dashboard/proto/resource_service.proto [hunk 1]
```diff
@@ -43,12 +43,19 @@ message ResourceCommand {
...
+    // Optional icon variant.
+    optional IconVariant icon_variant = 7;
     // Optional description of the command, to be shown in the UI.
     // Could be used as a tooltip. May be localized.
-    optional string display_description = 7;
+    optional string display_description = 8;
     // The state of the command. Controls whether the command is enabled, disabled,
     // or hidden in the UI.
-    ResourceCommandState state = 8;
+    ResourceCommandState state = 9;
```
Comment (drewnoakes) [inline src/Aspire.Hosting/Dashboard/proto/resource_service.proto:53]: Are we 100% sure no one has taken a dependency upon these field ids?

**Review by drewnoakes — APPROVED** [review]
Looks much better.

File: src/Aspire.Hosting/Dashboard/proto/resource_service.proto (hunk 1 above)
Comment (JamesNK) [reply src/Aspire.Hosting/Dashboard/proto/resource_service.proto:53]: Yes. They were added a couple of days ago.

**Pull request merged**